build.gradle

```diff
@@ -80,3 +79,10 @@ task ktlint(type: JavaExec, group: "verification") {
     main = "com.pinterest.ktlint.Main"
     args "app/src/**/*.kt"
 }
+
+task detektProjectBaseline(type: io.gitlab.arturbosch.detekt.DetektCreateBaselineTask) {
+    exclude("**/resources/**")
+    exclude("**/build/**")
+    exclude("**/tmp/**")
+    exclude(".*test.*")
```

Comment:
Is this a correct globing pattern?

Reply:
Changed to match original.

Reply:
I meant that `.*test.*` was a regexp, now it'g glob pattern where `.` means `.` but not one symbol. :)

Maybe we should change this to `**/test/**` to exclude test source. And rerun detekt to be sure that it matches everything we need.

Reply:
Done.